aws/ami/build_deb_ami.sh

```diff
@@ -62,6 +64,12 @@ while [ $# -gt 0 ]; do
             DOWNLOAD_ONLY=1
             shift 1
             ;;
+        "--dry-run")
+            echo "!!! Running in DRY-RUN mode !!!"
+            PACKER_SUB_CMD="validate"
+            shift 1
+            ;;
+        *)
```

Comment:
It's redundant - should be removed

Reply:
You meant the extra *)
Already found it myself.